typeclasses/baseobject.py

```diff
@@ -0,0 +1,19 @@
+from evennia import DefaultObject
```

Comment:
In addition, we don't need "typeclasses/baseobject", we already have "typeclasses/objects.py" with an Object class in it. Thing should inherit from that typeclasses.objects.Object and not DefaultObject in Evennia.